### DISCUSSION TITLE
Workflow to update files on fork

### DISCUSSION THREAD

**Discussion by tunisiano187** [community/community#38583] [Actions]
Hello evenyone,
I'm currently working on a template repository, and i would like to know if there is a solution to start a workflow based on forking the template?

The idea is to make it easy for the users to have the correct repository with their informations.
For example, when forking, i would like the workflow badge to be updated to the new repository, ...
Not hard with grep or powershell, so it would be easy to setup...

ewample
```yaml
on:
  fork:
```

Thank you.

**Comment by ossanna16** [comment]
Hi there @tunisiano187 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
